docs/guides/admin/docs/workflowoperationhandlers/amberscript-start-transcription-woh.md

```diff
@@ -14,13 +14,18 @@ an audio or video file to be transcribed and captioned.
 Parameter Table
 ---------------
 
-| configuration keys    | description                                                                | default      | example            |
-|-----------------------|----------------------------------------------------------------------------|--------------|--------------------|
-| source-tag            | A tag selecting the audio or video file to be sent for translation/transcription. | -     | engage-download    |
-| source-flavor         | A flavor selecting the audio or video file to be sent for translation/transcription. | -  | \*/themed           |
-| jobtype               | direct (automated, fast) or perfect (additional manual improvements, slow) | direct       | perfect            |
-| language              | The target language for transcription.                                     | en           | nl                 |
-| skip-if-flavor-exists | If this flavor already exists in the media package, skip this operation.   | captions/vtt | captions/timedtext |
+| configuration keys    | description                                                                              | default       | example                  |
+|-----------------------|------------------------------------------------------------------------------------------|---------------|--------------------------|
+| source-tag            | A tag selecting the audio or video file to be sent for translation/transcription.        | -             | engage-download          |
+| source-flavor         | A flavor selecting the audio or video file to be sent for translation/transcription.     | -             | \*/themed                |
+| jobtype               | direct (automated, fast) or perfect (additional manual improvements, slow).              | direct        | perfect                  |
+| language              | The assumed language for transcription.                                                  | en            | nl                       |
+| speaker               | The number of speakers in the recording.                                                 | 1             | 2                        |
+| transcriptiontype     | transcription, captions or translatedSubtitles                                           | transcription | captions                 |
+| glossary              | An ID of a custom glossary to use. See https://amberscript.github.io/api-docs/#glossary. | -             | 643966c3f3e91e0c96e9e060 |
```

Comment:
Behaviour of unset vs `""` should be documented here, and/or in the config.

Reply:
Properly explaining this would blow up the table; I will add a small section at the bottom, I hope that's okay.